proposals/2020/NMEC-Implementation/SLR-Example.xml

```diff
@@ -0,0 +1,490 @@
+                  <auc:StartTimestamp>2012-03-01T00:00:00</auc:StartTimestamp>
+                  <auc:IntervalFrequency>Day</auc:IntervalFrequency>
+                  <auc:IntervalReading>38.42</auc:IntervalReading>
+                </auc:TimeSeries>
+              </auc:TimeSeriesData>
+              <auc:LinkedPremises>
+                <auc:Building>
+                  <auc:LinkedBuildingID IDref="My-Fav-Building"/>
+                </auc:Building>
+              </auc:LinkedPremises>
+            </auc:Scenario>
+            <!-- Derived Model Scenario - Now 2 Models Defined -->
+            <auc:Scenario ID="Scenario-DerivedModel-2">
+              <auc:ScenarioType>
+                <auc:DerivedModel ID="Scenario-DM-StandardConditions">
+                  <auc:DerivedModelName>DerivedModel 1</auc:DerivedModelName>
+                  <auc:MeasuredScenarioID IDref="Scenario-Measured-2"/>
+                  <auc:Models>
+                    <!-- This begins our Model definition for the baseline period.
+                      (no changes compared to Model/@ID=Scenario-Baseline-DM-SLR) -->
+                    <auc:Model ID="Scenario-Baseline-DM-StandardConditions-SLR">
+                      <auc:StartTimestamp>2012-03-01T00:00:00</auc:StartTimestamp>
+                      <auc:EndTimestamp>2013-02-28T00:00:00</auc:EndTimestamp>
+                      <auc:DerivedModelInputs>
+                        <auc:IntervalFrequency>Day</auc:IntervalFrequency>
+                        <auc:ResponseVariable>
+                          <auc:ResponseVariableName>Electricity</auc:ResponseVariableName>
+                          <auc:ResponseVariableUnits>kWh</auc:ResponseVariableUnits>
+                          <auc:ResponseVariableEndUse>All end uses</auc:ResponseVariableEndUse>
+                        </auc:ResponseVariable>
+                        <auc:ExplanatoryVariables>
+                          <auc:ExplanatoryVariable>
+                            <auc:ExplanatoryVariableName>Drybulb Temperature</auc:ExplanatoryVariableName>
+                            <auc:ExplanatoryVariableUnits>Fahrenheit, F</auc:ExplanatoryVariableUnits>
+                          </auc:ExplanatoryVariable>
+                        </auc:ExplanatoryVariables>
+                      </auc:DerivedModelInputs>
+                      <auc:DerivedModelCoefficients>
+                        <auc:Guideline14Model>
+                          <auc:ModelType>2 parameter simple linear regression</auc:ModelType>
+                          <auc:Intercept>31433.86</auc:Intercept>
+                          <auc:Beta1>-288.1071</auc:Beta1>
+                        </auc:Guideline14Model>
+                      </auc:DerivedModelCoefficients>
+                      <!-- Model performance is summarized here.  -->
+                      <auc:DerivedModelPerformance>
+                        <auc:RSquared>0.69</auc:RSquared>
```

Comment:
I'm good for now with this. But do we want to make this generic, e.g.

```
<auc:preformance_variable>
  <auc:variable_name>RSquare</auc:variable_name>
  <auc:variable_value>0.69</auc:variable_value>
</auc:performance_variable>
```

For now more explicit is probably better.

Reply:
agree